xbmc/utils/test/TestURIUtils.cpp

```diff
@@ -600,3 +600,64 @@ TEST_F(TestURIUtils, UpdateUrlEncoding)
   EXPECT_FALSE(URIUtils::UpdateUrlEncoding(oldUrl));
   EXPECT_STRCASEEQ(newUrl.c_str(), oldUrl.c_str());
 }
+
+TEST_F(TestURIUtils, ServiceAvailabilityInTestEnvironment)
+{
+  URIUtils::IsOnLAN("ftp://some.where/foo.avi", LanCheckMode::ANY_PRIVATE_SUBNET);
+  URIUtils::IsHostOnLAN("ftp://some.where/foo.avi", LanCheckMode::ANY_PRIVATE_SUBNET);
```

Comment:
What is the point of the `IsHostOnLAN` check here and bellow with URLs instead of host names?